--- sysbench-0.4.12/acinclude.m4~	2009-03-14 14:35:36.000000000 +0100
+++ sysbench-0.4.12/acinclude.m4	2018-03-10 23:19:19.161781346 +0100
@@ -1,9 +1,7 @@
 dnl ---------------------------------------------------------------------------
 dnl Macro: AC_CHECK_MYSQLR
 dnl First check for custom MySQL paths in --with-mysql-* options.
-dnl If some paths are missing,  check if mysql_config exists. 
-dnl Then check for the libraries and replace -lmysqlclient with 
-dnl -lmysqlclient_r, to enable threaded client library.
+dnl If some paths are missing,  check if mysql_config exists.
 dnl ---------------------------------------------------------------------------
 
 AC_DEFUN([AC_CHECK_MYSQLR],[
@@ -51,7 +49,7 @@
     ac_cv_mysql_libs=`echo ${ac_cv_mysql_libs} | sed -e 's/.libs$//' \
                       -e 's+.libs/$++'`
     AC_CACHE_CHECK([MySQL libraries], [ac_cv_mysql_libs], [ac_cv_mysql_libs=""])
-    MYSQL_LIBS="-L$ac_cv_mysql_libs -lmysqlclient_r"
+    MYSQL_LIBS="-L$ac_cv_mysql_libs -lmysqlclient"
 fi
 
 # If some path is missing, try to autodetermine with mysql_config
@@ -82,8 +80,7 @@
         if test [ -z "$ac_cv_mysql_libs" ]
         then
             AC_MSG_CHECKING(MySQL linker flags)
-            MYSQL_LIBS=`${mysqlconfig} --libs | sed -e \
-            's/-lmysqlclient /-lmysqlclient_r /' -e 's/-lmysqlclient$/-lmysqlclient_r/'`
+            MYSQL_LIBS=`${mysqlconfig} --libs`
             AC_MSG_RESULT($MYSQL_LIBS)
         fi
     fi
